vm/systemSmartContracts/eei.go

```diff
@@ -729,6 +736,27 @@ func (host *vmContext) AddTxValueToSmartContract(value *big.Int, scAddress []byt
 	destAcc.BalanceDelta = big.NewInt(0).Add(destAcc.BalanceDelta, value)
 }
 
+// SetOwnerOperatingOnAccount will set the new owner, operating on the user account directly as the normal flow through
+// SC processor is not possible
+func (host *vmContext) SetOwnerOperatingOnAccount(arguments vm.SetOwnerArguments) error {
+	scAccount, err := host.userAccountsDB.LoadAccount(arguments.SCAddress)
```

Comment:
you do not need arguments.SCAddress here. Use host.scAddress instead. It is already equal with vmInput.RecipientAddress

Reply:
You only need the new owner address as argument

Reply:
Great! That will simplify a lot the function signature. :+1:

Reply:
done, simplified.